presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/FunctionCallProvider.java

```diff
@@ -62,6 +62,40 @@ public FunctionCall getExpectedValue(SymbolAliases aliases)
         return new FunctionCall(name, symbolReferences);
     }
 
+    public static ExpectedValueProvider<FunctionCall> windowFunctionCall(String name, List<String> args)
+    {
+        return new FunctionCallProvider(true, QualifiedName.of(name), false, toSymbolAliases(args));
+    }
+
+    public static ExpectedValueProvider<FunctionCall> functionCall(
+            String name,
+            boolean distinct,
+            List<PlanTestSymbol> args)
+    {
+        return new FunctionCallProvider(false, QualifiedName.of(name), distinct, args);
+    }
+
+    public static ExpectedValueProvider<FunctionCall> functionCall(String name, List<String> args)
+    {
+        return new FunctionCallProvider(QualifiedName.of(name), toSymbolAliases(args));
+    }
+
+    private static List<PlanTestSymbol> toSymbolAliases(List<String> aliases)
```

Comment:
Why on earth is this code duplicated from PlanMatchPattern?

Make it public there.